modules/pattern/pattern.service.js

```diff
@@ -137,14 +137,15 @@ class PatternsService {
       }).exec();
     }
 
-    const uploadResult = await uploadService.uploadFile(image[0]);
-    const images = uploadResult.fileNames;
-    const constructorImg = await uploadSmallImage(image[1]);
-    pattern.constructorImg = constructorImg;
-
-    if (!images && constructorImg) {
-      return Pattern.findByIdAndUpdate(id, pattern).exec();
+    if (image[0].file) {
+      const uploadResult = await uploadService.uploadFile(image[0]);
```

Comment:
if we use uploadSmallImage, so then can we create another function for large image?

Reply:
unfortunately using this function the page breaks